src/script/script.js

```diff
@@ -0,0 +1,402 @@
+          { className: "ArrowUp", value: "▲", symbol: false },
+          { className: "ShiftRight", value: "Shift", symbol: false },
+        ],
+        [
+          { className: "ControlLeft", value: "Ctrl", symbol: false },
+          { className: "MetaLeft", value: "Win", symbol: false },
+          { className: "AltLeft", value: "Alt", symbol: false },
+          { className: "Space", value: "", symbol: false },
+          { className: "AltRight", value: "Alt", symbol: false },
+          { className: "ContextMenu", value: "", symbol: false },
+          { className: "ArrowLeft", value: "◄", symbol: false },
+          { className: "ArrowDown", value: "▼", symbol: false },
+          { className: "ArrowRight", value: "►", symbol: false },
+          { className: "ControlRight", value: "Ctrl", symbol: false }
+        ]
+      ];
+  }
+
+  setRootElement() {
+    this.rootElement = document.createElement('div');
+    this.rootElement.classList.add('virtual-keyboard-wrapper');
+    document.body.prepend(this.rootElement);
+  }
+  ///////////////////////////////////////////////////////////////
+  generateHeader() {
+    if (!this.rootElement) this.setRootElement();
+    this.header = document.createElement('div');
+    this.header.classList.add('header-wrapper');
+    this.header.innerHTML = `
+      <h1 class="header">RS-School task Virtual Keyboard</h1>
+      <p class="description">Клавиатура создана в операционной системе Windows</p>
+      <p class="description">Переключения языка клавиатурой: левыe ctrl + alt</p>
+      <p class="description">Переключение языка мышкой: кнопка с иконкой глобуса</p>`;
+    this.rootElement.append(this.header);
+  }
+
+  generateTextArea() {
+    if (!this.rootElement) this.setRootElement();
+    let textArea = document.createElement('div');
+    textArea = document.createElement('div');
+    textArea.classList.add('textarea-wrapper');
+    textArea.innerHTML = `
+      <textarea class="textarea" id="textarea" rows="5" cols="50" autofocus></textarea>`;
+    this.rootElement.append(textArea);
+    this.textArea = document.querySelector('#textarea');
+    this.textArea.value = '';
+  }
+
+  generateKeyboard() {
+    if (!this.rootElement) this.setRootElement();
+
+    const rows = this.data || this.defaultSet;
+
+    this.keyBoard = document.createElement('div');
+    this.keyBoard.classList.add('keyboard');
+    rows.forEach(elem => {
+      this.keyBoard.append(this.generateKeyboardRow(elem));
+    });
+    this.rootElement.append(this.keyBoard);
+    this.keyBoard = document.querySelector('.keyboard');
+    this.keys = document.querySelectorAll('.symbol');
+    this.languageButton = document.querySelector('.ContextMenu span');
+
+    this.pressed = new Set();
+
+    this.textArea.addEventListener("click", () => {
+      this.currentPosition = this.getCaretPosition();
+    })
+
+    document.addEventListener("keydown", (event) => {
+      if (event.code === "F5") return;
```

Comment:
It`s better to use direct condition (if (event.code !== "F5") ... )

Reply:
Fixed